README.md

```diff
@@ -32,610 +34,107 @@ For more information refer to [IntelliJ IDEA documentation](https://www.jetbrain
-[in the source code](https://github.com/async-profiler/async-profiler/blob/v3.0/src/arguments.cpp#L44).
-`asprof` actually converts command line arguments to that format.
-
-For instance, `-e wall` is converted to `event=wall`, `-f profile.html`
-is converted to `file=profile.html`, and so on. However, some arguments are processed
-directly by `asprof`. E.g. `-d 5` results in 3 actions:
-attaching profiler agent with start command, sleeping for 5 seconds,
-and then attaching the agent again with stop command.
-
-## Multiple events
-
-It is possible to profile CPU, allocations, and locks at the same time.
-Instead of CPU, you may choose any other execution event: wall-clock,
-perf event, tracepoint, Java method, etc.
-
-The only output format that supports multiple events together is JFR.
-The recording will contain the following event types:
- - `jdk.ExecutionSample`
- - `jdk.ObjectAllocationInNewTLAB` (alloc)
- - `jdk.ObjectAllocationOutsideTLAB` (alloc)
- - `jdk.JavaMonitorEnter` (lock)
- - `jdk.ThreadPark` (lock)
-
-To start profiling cpu + allocations + locks together, specify
+In a typical use case, profiling a Java application is just a matter of a running `asprof` with a PID of a
+running Java process.
 ```
-asprof -e cpu,alloc,lock -f profile.jfr ...
+$ asprof -d 30 -f /tmp/flamegraph.html <PID>
 ```
-or use `--alloc` and `--lock` parameters with the desired threshold:
-```
-asprof -e cpu --alloc 2m --lock 10ms -f profile.jfr ...
-```
-The same, when starting profiler as an agent:
-```
--agentpath:/path/to/libasyncProfiler.so=start,event=cpu,alloc=2m,lock=10ms,file=profile.jfr
-```
-
-## Flame Graph visualization
-
-async-profiler provides out-of-the-box [Flame Graph](https://github.com/BrendanGregg/FlameGraph) support.
-Specify `-o flamegraph` argument to dump profiling results as an interactive HTML Flame Graph.
-Also, Flame Graph output format will be chosen automatically if the target filename ends with `.html`.
-
-```
-$ jps
-9234 Jps
-8983 Computey
-$ asprof -d 30 -f /tmp/flamegraph.html 8983
-```
-
-[![Example](https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.png)](https://htmlpreview.github.io/?https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.html)
-
-## Profiler Options
-
-`asprof` command-line options.
-
-* `start` - starts profiling in semi-automatic mode, i.e. profiler will run
-  until `stop` command is explicitly called.
-
-* `resume` - starts or resumes earlier profiling session that has been stopped.
-  All the collected data remains valid. The profiling options are not preserved
-  between sessions, and should be specified again.
-
-* `stop` - stops profiling and prints the report.
 
-* `dump` - dump collected data without stopping profiling session.
+[![Sample FlameGraph](https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.png)](https://htmlpreview.github.io/?https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.html)
```

Comment:
`flamegraph.png/html` existed in `demo` folder for the only purpose of referring from README.
They can be moved to `.assets` just like as other documentation images.

Reply:
[adc296d](https://github.com/async-profiler/async-profiler/pull/1029/commits/adc296d22ffe7bcf227f89b9b92e799a2f72f2f0)